tests/samples/test_api.py

```diff
@@ -857,7 +685,9 @@ async def test_finalize(
         with pytest.raises(ResourceNotFoundError):
             await get_data_from_app(client.app).uploads.get(1)
 
-        assert not (await get_row_by_id(pg, SQLSampleReads, 1)).upload
+        assert not (
+            await get_row_by_id(data_layer.samples._pg, SQLSampleReads, 1)
+        ).upload
```

Comment:
Doing assertions against databases should be in data layer tests. Am I missing something here?